wetmap/src/components/newModals/shopModal/view.tsx

```diff
@@ -75,77 +43,72 @@ export default function ShopModalView(props: ShopModelViewProps) {
               className="btn-lg"
-              onClick={() => fileUploaderRef?.current?.click?.()}
-            />
+              onClick={() => {}}
+              />
+            )}  
           </div>
         </WavyModalHeader>
         <div className="ml-6">
           <div className="stack-4">
             <div>
               <div className="d-flex">
-                <h1 className="mb-0">{props?.shopModelName}</h1>
-                <div>
-                  <Icon
-                    name="flag"
-                    fill="maroon"
-                    width="30px"
-                    onClick={() => window.location = `mailto:DiveGo2022@gmail.com?subject=Reporting%20issue%20with%20Dive%20Site:%20"${selectedDiveSite.SiteName}"%20at%20Latitude:%20${selectedDiveSite.Latitude}%20Longitude:%20${selectedDiveSite.Longitude}&body=Type%20of%20issue:%0D%0A%0D%0A%0D%0A%0D%0A1)%20Dive%20site%20name%20not%20correct%0D%0A%0D%0A(Please%20provide%20correct%20dive%20site%20name%20and%20we%20will%20correct%20the%20record)%0D%0A%0D%0A%0D%0A%0D%0A2)%20Dive%20site%20GPS%20coordinates%20are%20not%20correct%0D%0A%0D%0A(Please%20provide%20a%20correct%20latitude%20and%20longitude%20and%20we%20will%20update%20the%20record)`}
-                  />
-                </div>
+                <h1 className="mb-0">{props?.diveShop.orgName}</h1>
               </div>
+            </div>
 
-              {/* <div className="d-flex">
-                {'Added by: '}
-                <a href="#">{props?.diveSite?.newusername}</a>
-              </div> */}
+            <div className="panel border-none">
+              <div className="panel-body">
+                <PlainTextInput
+                  placeHolder={`A little about ${props?.diveShop?.orgName}`}
+                  content={props?.diveShop?.diveShopBio || ''} // diveShopBio doesn't exist yet.
+                  readOnly={!props?.isPartnerAccount}
+                  onSubmit={props?.onDiveShopBioChange}
+                />
+              </div>
             </div>
           </div>
         </div>
         <div className="ml-6">
           <h3 class="text-left">{props?.shopDescription}</h3>
         </div>
       </div>
-      <div className="col-6">
-        <h3>Offered Diving Trips</h3>
-        <div style={{ marginTop: '3%', width: '100%', borderRadius: 15 }}>
-          <div className="container5">
-            {itineraryList
-            && itineraryList.map((itinerary) => {
+      <div className="col-6 panel border-none full-height">
+        <div className="panel-header">
+          <h3>Offered Diving Trips</h3>
+          {props?.isPartnerAccount && (
+            <Button className="mt-2 btn-lg">
+                Add diving event
+            </Button> 
+          )}     
+        </div>
+        <div className={`${style.itineraryList}`}>
+          {props?.itineraryList// in the future, if itineraryList is not empty, render a loading spinner
+            && props?.itineraryList.map((itinerary) => {
```

Comment:
itinerary list is NOT a string, you'll have to figure out what type it actually should be, hint: itinerary needs to be a type as well so you'll need to define an type for it : type Itinerary = ......